docs/developers/packaging.md

```diff
@@ -255,7 +256,7 @@ Very fun! So where do all these packages live?
 | `constructor`      | [jaimergp/constructor @ `menuinst-cep`][9]      | [jaimergp-forge/constructor-feedstock][12]         |
 | `conda-standalone` | _Same as feedstock_.                            | [conda-forge/conda-standalone-feedstock PR#21][13] |
 | `conda`            | [jaimergp/conda @ `cep-menuinst`][10]           | [jaimergp-forge/conda-feedstock][14]               |
-| `menuinst`         | [jaimergp/menuinst @ `cep`][11]                 | [jaimergp-forge/menuinst-feedstock][15]            |
+| `menuinst`         | [conda/menuinst @ `cep-devel`][11]                 | [jaimergp-forge/menuinst-feedstock][15]            |
 
 
 Most of the forks live in `jaimergp`'s account, under a non-default branch. They are published
```

Comment:
the next line mentions branch `master`, should that be `main`?

Reply:
Good point. Some feedstocks have main, others have master. I'll add a commit.